shared/nm-utils/nm-shared-utils.c

```diff
@@ -2505,6 +2505,59 @@ _nm_utils_unescape_plain (char *str, const char *candidates, gboolean do_strip)
 	return str;
 }
 
+char *
+nm_utils_str_simpletokens_extract_next (char **p_line_start)
+{
```

Comment:
What the functions does is not clear from the name, can you add a comment?

Reply:
I did not, but the parsing/escaping here needs some further improvements. I will do that in a follow-up.